shell/protoc.sh

```diff
@@ -101,10 +101,10 @@ for import in $imports; do
   default_args+=("--proto_path=$import_path/$path")
   import_paths+=("$import_path/$path")
 
-  # Check to see if we already have this version locally and skip cloning
-  # if we already do.
+  # Check to see if we already have this version locally and remove
+  # to allow a fresh clone if we already do
```

Comment:
You're just going to remove the terminal period? Dirty.

Reply:
```suggestion
  # to allow a fresh clone if we already do.
```